v8/orgmode/init.el

```diff
@@ -111,6 +111,13 @@ contextual information."
             (pygmentize (downcase lang) (org-html-decode-plain-text code)))
         code-html))))
 
+;; Export internal links with custom link type
+(defun org-custom-internal-link-url-export (path desc format)
+  (cond
+   ((eq format 'html)
+    (format "<a href=\"%s/index.html\">\"%s\"" path desc))))
```

Comment:
Don’t include `index.html` in those links. It’s not necessary, and we allow changing the filename.

Reply:
Ah thanks didn't know that. just starting with nikola